doc/developer-guide/api/types/TSSslSession.en.rst

```diff
@@ -34,7 +34,6 @@ Synopsis
 
    .. member:: size_t len
 
-   .. member:: char bytes[TS_SSL_MAX_SSL_SESSION_ID_LENGTH]
```

Comment:
This seems unrelated. Did you have to do this to make docs happy for some reason? If so, we should make it a separate PR. I'd like to know what introduced the change in behavior...

Reply:
Yeh, this is my bad, I gave him a bad patch. This should not be done.

Reply:
Gotcha.